### PR TITLE
[319] Cookie banner and page

### PR DESCRIPTION
**Pull request by AbigailMcP**
## Changes in this PR

- Adds cookies banner
- Adds cookie policy page with link from footer
- Adds accessibility link which doesn't go anywhere for now - this was just to see how multiple links from the footer will look. I can remove if needs be

### CODE REVIEW

**Comment by claire-hughez** [comment]
[Create cookies policy and banner](https://trello.com/c/9qOCGIST/319-create-cookies-policy-and-banner)

File: layouts/base.html [hunk 1]
```diff
@@ -117,12 +119,20 @@
     <div class="govuk-width-container">
       <div class="govuk-footer__meta">
         <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+          <ul class="govuk-footer__inline-list govuk-!-margin-bottom-5">
+            <li class="govuk-footer__inline-list-item">
+              <a href="<%= @base_url %>/#" class="govuk-link govuk-footer__link">Accessibility</a>
```
Comment (claire-hughez) [inline layouts/base.html:124]:
```suggestion
              <a href="<%= @base_url %>/#" class="govuk-link govuk-footer__link">Accessibility statement</a>
```

**Comment by claire-hughez** [comment]
Looks great! A few notes
1 - Agree to just leave Accessibility statement there for now, we'll need it soon.
2 - The buttons on design.education.gov.uk are green - is there a particular design reason these are navy, or I reckon we should stick with green as the default? 
3 - I've made some suggested content changes, if Ashleigh is happy to, I think we should change to this.

File: content/cookie-policy.html.erb [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        <p>
+          Google is not allowed to use or share our analytics data with anyone.
+        </p>
+        <p>
+          Google Analytics stores anonymised information about: 
+        </p>
+
+        <ul>
+          <li>
+            how you got to these sites
```
Comment (claire-hughez) [inline content/cookie-policy.html.erb:70]:
```suggestion
            how you got to this service
```

File: content/cookie-policy.html.erb [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          </li>
+          <li>
+            the pages you visit and how long you spend on each page
+          </li>
+          <li>
+            what you click on while you’re visiting these sites
```
Comment (claire-hughez) [inline content/cookie-policy.html.erb:76]:
```suggestion
            what you click on while you’re visiting this service
```

**Comment by AbigailMcP** [comment]
> Looks great! A few notes 1 - Agree to just leave Accessibility statement there for now, we'll need it soon. 2 - The buttons on design.education.gov.uk are green - is there a particular design reason these are navy, or I reckon we should stick with green as the default? 3 - I've made some suggested content changes, if Ashleigh is happy to, I think we should change to this.

Thanks! I'll check with Lauren about the buttons. I copied the ones we use across the service, but happy to change to green!

File: layouts/base.html (hunk 1 above)
Comment (ashleighk27) [reply layouts/base.html:124]: Agree with this one, it mirrors the rest of GOV!

**Comment by ashleighk27** [comment]
Reviewed and agree with your edits Claire :)

**Review by mark-pkgflow — APPROVED** [review]

**Pull request merged**